### ISSUE TITLE
Incorrect values in contrast coefficients table

### ISSUE THREAD

**Issue by benminder** [gamlj/gamlj#205]
When conducting a one-way ANOVA on a variable with 4 levels, the verbal descriptions of the contrasts aren't aligned with the contrast coefficients listed in the table. This issue affects versions 2.x and 3.x.

It looks like the contrasts coefficients for Simple and Deviation coding are mixed up with each other:

Simple: 

<img width="468" alt="Simple" src="https://github.com/gamlj/gamlj/assets/61423773/5c2efd47-9b81-4724-903f-18ef05515269">

Deviation:

<img width="468" alt="Deviation" src="https://github.com/gamlj/gamlj/assets/61423773/e81b072f-c08d-4d28-8a81-3500f466b431">

**Comment by benminder** [comment]
Yes, you are right, this was my mistake. I'm embarrassed to say that I bothered you with the same mistake 2 years ago and completely forgot about it🤦

**Comment by mcfanda** [comment]
On the contrary. If you didn't forget, I did not have the chance to check it again. Open source software critically depends on the community's input. Please  keep spotting any issue that might be there, false positive are welcome
